docs/examples/viz_network_animated_billboard.py

```diff
@@ -0,0 +1,282 @@
+"""
+=======================================================
+Visualize Networks using Billboard (Animated version)
+=======================================================
+
+The goal of this demo is to show how to visualize a
+complex network using billboards and simple interaction
+and use a force directed algorithm to layout the network.
+Hovering over nodes will show node index.
+
+"""
+
+###############################################################################
+# First, let's import some useful modules and functions
+
+import math
+import random
+import time
+from collections import Counter
+from os.path import join as pjoin
+import numpy as np
+import helios
+import vtk
+import vtk.util.numpy_support as vtknp
+from fury import actor, window, ui, colormap as cmap
+from fury.shaders import load as shader_load
+import fury.primitive as fp
+from fury.utils import (get_actor_from_polydata, numpy_to_vtk_colors,
+                        set_polydata_triangles, set_polydata_vertices,
+                        set_polydata_colors, colors_from_actor,
+                        vertices_from_actor, update_actor, compute_bounds,
+                        vtk_vertices_from_actor, vtk_array_from_actor)
```

Comment:
pep8: Can you remove all the unused import / function?

Reply:
Done